### PR TITLE
Implement feedback

### PR DESCRIPTION
**Pull request by veryobinna**
#### What does this PR do?
This PR improves the user interface of the login page and the home page and adds more component tests.

#### Description of Task to be completed?
Implement the feedback given during project defense

### CODE REVIEW

File: app/app.jsx [hunk 1]
```diff
@@ -33,7 +33,6 @@ function checkAuth(nextState, replace) {
 ReactDOM.render(
   <Router history={hashHistory}>
```
Comment (iAmao) [inline app/app.jsx:34]: I don't think you want to use hashHistory in production

File: app/components/Newsfeeds.jsx [hunk 2]
```diff
@@ -8,6 +8,7 @@ export default class Newsfeed extends React.Component {
     this.handleArticleChange = this.handleArticleChange.bind(this);
     this.state = {
```
Comment (iAmao) [inline app/components/Newsfeeds.jsx]: if the properties of this object can fit on a line there is no need to break them

File: app/components/Newsfeeds.jsx [hunk 3]
```diff
@@ -21,19 +22,21 @@ export default class Newsfeed extends React.Component {
...
       return (
         <div key={index} className="col-md-4 col-sm-6">
-          <div className="thumbnail">
+          <div className="thumbnail" href={article.url}>
             <img src={article.urlToImage} alt="" />
             <div className="caption">
-              <h3>{article.title}</h3>
-              <small>{article.publishedAt} </small>
+              <a href={article.url}> <h4>{article.title}</h4> </a>
+
```
Comment (iAmao) [inline app/components/Newsfeeds.jsx]: redundant empty line

**Comment by iAmao** [comment]
- Fix your travis build
- remove your bundled file from repo

- remove unnecessary lines in your webpack file
- No need to break a jsx tag if the attributes can be contained in a file
- in your newsStore.js you might want to move these
```
_articles = action.news.articles;
 NewsStore.emitChange();
```
to functions of their own
-  It is redundant to have a emitChange() method in your NewsStore. you can just do something like
```
NewsStore.emit(CHANGE_EVENT)
```
- I can't see a Review app link
- You have a User folder. Whats it for?  From the looks I'm guessing its a helper class. Please rename with something more descriptive

- in your store you might want to refactor to es6 something along the lines of:

```
class NewsStore extends EventEmitter {

  constructor() {
    super();
    this.articles;
    this.sources;
  }

 getNews() {
    return this._articles;
  }
  displaySource() {
    return this._sources;
  }

  emitChange() {
    this.emit(CHANGE_EVENT);
  }

  addChangeListener(callback) {
    this.on(CHANGE_EVENT, callback);
  }

  removeChangeListener(callback) {
    this.removeListener(CHANGE_EVENT, callback);
  }
}
```

Overall, this is a strong attempt. Very clean. Nice

**Comment by veryobinna** [comment]
Thank you so much.I will implement all the corrections.

File: app/app.jsx (hunk 1 above)
Comment (iAmao) [reply app/app.jsx:34]: try browserHistory

**Pull request merged**